app/main/views/marketplace.py

```diff
@@ -269,6 +269,13 @@ def get_brief_by_id(framework_slug, brief_id):
     application_url = "/sellers/opportunities/{}/responses/create".format(brief['id'])
     add_case_study_url = None
 
+    # pass application status to brief
+    profile_application_status = None
+    if current_user.is_authenticated and current_user.role == 'supplier':
+        profile_application_id = current_user.application_id
```

Comment:
I don't think type status changes the flow on brief application, you are either 'approved' and go to the application or you are submitted and get a 'assessment is underway' msg.